scripts/update-tester-image.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
```

Comment:
Do we want to leave this here?

Reply:
That's being used in all of these scripts right now, these scripts themselves are temporary until #10.